UI/GameInfoCache.cpp

```diff
@@ -405,6 +405,10 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 					std::lock_guard<std::mutex> lock(info_->lock);
 					info_->paramSFO.ReadSFO(sfoData);
 					info_->ParseParamSFO();
+					if (info_->id.empty() || info_->id == "UCJS10041") {
+						info_->id = g_paramSFO.generateFakeID(gamePath_);
```

Comment:
Any place actually requires it and isn't fine with empty string? There's really nothing to base it on in homebrew, so that would be completely hardcoded and made up number.